packages/vsx-registry/src/browser/vsx-extension-commands.ts

```diff
@@ -36,6 +36,13 @@ export namespace VSXExtensionsCommands {
         label: nls.localizeByDefault('Install from VSIX') + '...',
         dialogLabel: nls.localizeByDefault('Install from VSIX')
     };
+    export const INSTALL_VSIX: Command = {
```

Comment:
nit: I think we could use `Command.toDefaultLocalizedCommand` here.

Reply:
Not sure if we should, I need to check it. Thanks!

Reply:
I see, `toDefaultLocalizedCommand` does the job for me. Thanks!